lib/ansible/modules/cloud/amazon/dms_replication_subnet_group.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
please use short notation of the GPL header

Reply:
same